README.md

```diff
@@ -167,8 +167,8 @@ Once the instance is started, you can start a session:
 If you prefer a command-line tool, you can start a session with a recent [AWS CLI](https://aws.amazon.com/cli/) and the [session-manager-plugin](https://docs.aws.amazon.com/systems-manager/latest/userguide/session-manager-working-with-install-plugin.html).
 Then you'd be able to start a session using only your instance ID, like this:
 
-```shell
-aws ssm start-session --target INSTANCE_ID
+```shell 
```

Comment:
nit: there is an extra space here, is it needed?

Reply:
I don't see any extra space